src/components/common/toast/toast.styles.tsx

```diff
@@ -0,0 +1,6 @@
+import { Color, Text, Toast } from "@lifesg/react-design-system";
```

Comment:
- remove unused `Color` and `Text` imports
- use sub-directory import to reduce import size